protostar/cheatable_starknet/cheatcodes/cairo_cheatcode.py

```diff
@@ -1,12 +1,65 @@
-from abc import ABC
-from typing import TYPE_CHECKING
+from abc import ABC, abstractmethod
+from dataclasses import dataclass
+from typing import TYPE_CHECKING, Any, Callable, Union
+
+from starknet_py.cairo.felt import encode_shortstring
 
 from protostar.cairo import HintLocal
+from protostar.cheatable_starknet.cheatable_exception import CheatableException
 
 if TYPE_CHECKING:
     from protostar.cheatable_starknet.cheaters import CairoCheaters
 
+Felt = int
```

Comment:
```suggestion
```
It is misleading, this is int not a field element

Reply:
[remove felt](https://github.com/software-mansion/protostar/pull/1404/commits/10ef1c98c597b049f223197328e8fca2c052ab15)